okhttp-hpacktests/pom.xml

```diff
@@ -8,7 +8,7 @@
   <parent>
     <groupId>com.squareup.okhttp3</groupId>
     <artifactId>parent</artifactId>
-    <version>3.3.0-SNAPSHOT</version>
```

Comment:
It’s currently disabled due to its dependence on git submodules. We could just snapshot the sample data though.